src/components/Home/Home.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCountries } from '../../redux/contries/countries';
+import CountryList from '../CountryList/CountryList';
+import style from './Home.module.css';
+
+function Home() {
+  const dispatch = useDispatch();
+  const countries = useSelector((state) => state.countries);
+  const [filter, setFilter] = useState(countries);
+  const filterCountries = (e) => {
+    const target = e.target.value;
+    if (target === 'all') {
+      setFilter(countries);
+    } else {
+      setFilter(countries.filter((el) => el.region === target));
+    }
+  };
+
+  useEffect(
+    () => { dispatch(getCountries()); setFilter(null); },
+    [dispatch],
+  );
+  return (
+    <div data-testid="home-container">
+      <div className={style.dropDown}>
+        <select
+          name="drop-down"
+          onChange={filterCountries}
+        >
+          <option value="all">All</option>
+          <option value="Asia">Asia</option>
+          <option value="Africa">Africa</option>
+          <option value="Americas">America</option>
+          <option value="Europe">Europe</option>
+          <option value="Oceania">Oceania</option>
+        </select>
+      </div>
+      <CountryList countries={filter || countries} />
+    </div>
+  );
+}
+
+export default Home;
```

Comment:
- [ ] Good job so far with your project it looks great and works as expected :clap:  But we have an issue with your design :cry: As you check on the project requirement, we expect you to work on mobile design and we can see some differences between your design and the original design. Please try to make your design similar to the original design for both the home and details page as a developer it is important to pay attention to details :+1: 

#### Figma design
![metrics_app](https://user-images.githubusercontent.com/6615394/195218382-33f7f77a-f88f-41ac-a8ce-795156a3a495.jpg)

#### You design
![Screenshot from 2022-10-11 23-35-47](https://user-images.githubusercontent.com/6615394/195218428-be42442d-78ff-4173-92e8-e44c50f28316.png)
![Screenshot from 2022-10-11 23-34-13](https://user-images.githubusercontent.com/6615394/195218432-1622589d-1258-4340-9469-23f9442622f9.png)

- [ ] Please make sure to follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), especially the layout: composition and space between elements as request for this project :+1: